tests/func/test_import_url.py

```diff
@@ -46,7 +46,7 @@ def test(self):
 
 
 class TestFailedImportMessage(TestDvc):
-    @patch("dvc.repo.imp_url.urlparse")
+    @patch("dvc.repo.imp_url.URLInfo")
```

Comment:
```suggestion
    @patch("dvc.repo.imp_url.URLInfo", side_effect=Exception("message"))
```
And drop side effect handling in func body.